cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesImport.java

```diff
@@ -328,6 +330,11 @@ private void copyStream(ReadOnlyDataSource from, DataSource to, String fromName,
             ParameterType.STRING,
             "Path of ID mapping file",
             null);
+    private static final Parameter ID_MAPPING_FILE_NAMING_STRATEGY_PARAMETER = new Parameter(
+            ID_MAPPING_FILE_NAMING_STRATEGY,
+            ParameterType.STRING,
+            "Configure what type of naming strategy you want to use for the provided ID mapping file",
+            "cgmes");
```

Comment:
In order to prevent breaking change, I think the default parameter should be "identity"

Reply:
Done. The unit test had to be adjusted to avoid using identity naming strategy when there is a mapping file.